src/app/components/chat/room.ts

```diff
@@ -41,9 +40,6 @@ export class ChatRoom {
 	}
 }
 
-export function getChatRoomTitle(room: ChatRoom, chat: ChatClient) {
-	return room.members
-		.filter(member => member.id !== chat.currentUser?.id)
-		.map(member => member.display_name)
-		.join(', ');
+export function getChatRoomTitle(room: ChatRoom) {
+	return room.members.map(member => member.display_name).join(', ');
```

Comment:
Why is this changed? I'd think we want to not show the logged in user name in room titles.

Reply:
Because it means the room title is blank when the room is initially created.